plugins/services/src/js/pages/task-details/TaskLogsTab.js

```diff
@@ -168,13 +178,11 @@ class TaskLogsTab extends mixin(StoreMixin) {
     const {task} = this.props;
     const {subscriptionID} = this.state;
 
-    // Stop tailing before fetching preiovus logs
-    SystemLogStore.stopTailing(this.state.subscriptionID);
-    // Fetch two pages previous log entries to gain more leverage to explore
+    // Fetch a full page previous log entries to gain more leverage to explore
```

Comment:
I notice the value of the `PAGE_ENTRY_CONT` doubled, but why are we doubling it there and not here anymore?